content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -32,7 +32,7 @@ see [CronJob](/docs/concepts/workloads/controllers/cron-jobs/).
 
 ## Running an example Job
 
-Here is an example Job config.  It computes π to 2000 places and prints it out.
+Here is an example Job config.  It computes value of π upto 2000 places and prints it out.
```

Comment:
```suggestion
Here is an example Job config.  It computes π to 2000 places and prints it out.
```
I prefer the original as it adheres to the [Kubernetes style guide for simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)